point_cloud_transport/src/publisher_plugin.cpp

```diff
@@ -38,10 +38,12 @@
 namespace point_cloud_transport
 {
 
-PublisherPlugin::EncodeResult PublisherPlugin::encode(const sensor_msgs::msg::PointCloud2 & raw)
+PublisherPlugin::EncodeResult PublisherPlugin::encode(const sensor_msgs::msg::PointCloud2 & /*raw*/)
 const
 {
-  return this->encode(raw);
+  // TODO(ahcorde): Review this, infinite recursion
+  // return this->encode(raw);
```

Comment:
This is concerning, we should probably understand what is happening here.

Reply:
I'm getting this error message:

```bash
 warning: all paths through this function will call itself [-Winfinite-recursion]
```